src/styles/questionDetails.scss

```diff
@@ -25,6 +25,7 @@
   font-size: large;
   cursor: pointer;
   transition: all 0.5s ease-in-out;
+  border-radius: 4px;
```

Comment:
I like that you have r all the borders a radius. We should work on an SCSS variables and mixins file to easily manage changes like this.